packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/ImageResultTask.kt

```diff
@@ -27,8 +28,10 @@ open class ImageResultTask(
   private val uri: Uri,
   private val contentResolver: ContentResolver,
   private val fileProvider: FileProvider,
+  private val isEdited: Boolean,
```

Comment:
I'm passing [`pickerOptions.isAllowsEditing`](https://github.com/expo/expo/blob/2607ac767410d2ae72dabe53d682245024bbd3f1/packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt#L359) to constructor to determine if photo is edited to prevent copying exif data from wrong image which could be possible in case of crash.